pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -308,6 +309,11 @@ let
       # To fix the build as we don't provide libffi_pic.a
       # (ld.lld: error: unable to find library -l:libffi_pic.a):
       use_system_libffi = true;
+    } // optionalAttrs (chromiumVersionAtLeast "115") {
```

Comment:
```suggestion
    } // lib.optionalAttrs (chromiumVersionAtLeast "115") {
```

Evaluation issue.

Reply:
Fixed, made this conditional for 115+ only later and didn't rebuild.